src/brsTypes/components/Timespan.ts

```diff
@@ -79,16 +83,12 @@ export class Timespan extends BrsComponent implements BrsValue {
             returns: ValueKind.Int32,
         },
         impl: (_: Interpreter, date: BrsString) => {
-            let dateAsSeconds;
-            let now = Date.now();
-            let dateToParse = luxon.DateTime.fromISO(date.value, { zone: "utc" });
-
-            if (dateToParse.isValid) {
-                dateAsSeconds = Math.round((Date.parse(dateToParse.toISO()) - now) / 1000);
-            } else {
-                dateAsSeconds = 2077252342;
+            let dateAsSeconds = 2077252342;
```

Comment:
Should this be removed?

Reply:
No, that's the default `fault` value Roku returns when an invalid parser date is sent.

Reply:
ah. okay cool.